lib/g_searcher_web/controllers/report_controller.ex

```diff
@@ -3,17 +3,20 @@ defmodule GSearcherWeb.ReportController do
 
   alias GSearcher.Reports
   alias GSearcherWeb.DashboardView
+  alias GSearcherWeb.Validators.CreateReport, as: ReportParams
```

Comment:
I might prefer just

```suggestion
  alias GSearcherWeb.Validators.CreateReportParams
```

or

```suggestion
  alias GSearcherWeb.Validators.ReportParams
```

to avoid confusion, what do you think? 🤔